io.github.MrPiggy92.ScriptedJourneys.json

```diff
@@ -0,0 +1,36 @@
+{
+  "id": "io.github.MrPiggy92.ScriptedJourneys",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "22.08",
```

Comment:
Can you use 24.08? 22.08 will be EOL by the end of this month.

Reply:
when i change it to 24.08, it says Running appstream-compose
bwrap: execvp appstream-compose: No such file or directory
Error: ERROR: appstream-compose failed: Child process exited with code 1

Reply:
Can you do the change here so that we can test? That is an error on your end, you are likely missing appstreamcli package.